js/utils/ajax.js

```diff
@@ -19,16 +19,43 @@
  */
 
 define([
-  'jquery'
-], function($) {
+  'jquery',
+  'token-utils',
+  'user-utils'
+], function($, token, user) {
+  function extractDomain(url) {
+    return url.split('/').slice(0, 3).join('/');
+  }
+
   $(document).ajaxError(function(event, jqueryXHR, error, errorThrown) {
+    var i, cluster,
+      currentCluster = window.cluster(),
+      isCurrentCluster = currentCluster && error.url.indexOf(currentCluster.api.url) > -1;
+
+    // find cluster concerned by current error
+    for (i in window.clusters) {
+      cluster = window.clusters[i];
+      if (cluster.api.url === extractDomain(error.url)) {
+        break;
+      }
+    }
+
     if (!jqueryXHR.status && !(error.url.indexOf('/authentication') > -1)) {
-      console.log(JSON.stringify(event), JSON.stringify(jqueryXHR), JSON.stringify(error), JSON.stringify(errorThrown));  // eslint-disable-line no-console
-      $('#flash .alert').append($('<p>').text('Error : unable to perform authentication'));
-      $('#flash').show();
+      // case when error status is not 403 (FORBIDDEN)
```

Comment:
The comment might be wrong since there is no check for != 403 here. Can you please explain in comment in what case the expression (!jqueryXHR.status && !(error.url.indexOf('/authentication') > -1)) is true.